ext/search_kit/Civi/Api4/Action/SearchDisplay/AbstractRunAction.php

```diff
@@ -177,7 +177,7 @@ private function getValue($key, $data, $rowIndex) {
       default:
         if (!empty($data[$key])) {
           $item = $this->getSelectExpression($key);
-          if ($item['expr'] instanceof SqlField && $item['fields'][$key]['fk_entity'] === 'File') {
+          if ($item['expr'] instanceof SqlField && isset($item['fields'][$key]) && $item['fields'][$key]['fk_entity'] === 'File') {
```

Comment:
this was triggering a notice error not sure if its something specific to my unit test or not but seemed a safe enough thing to do